Source/PeleLMeX_Setup.cpp

```diff
@@ -330,26 +351,54 @@ PeleLM::readParameters()
 
   // -----------------------------------------
   // diffusion
+  ParmParse pptrans("transport");
+  pptrans.query("use_soret", m_use_soret);
   pp.query("use_wbar", m_use_wbar);
-  pp.query("unity_Le", m_unity_Le);
```

Comment:
Can you leave in a query for `unity_Le` abort if it gets turned on with the message "unity_Le is depracated, used fixed_Le instead"? That would prevent anyone currently using the functionality from experiencing unexpected behavior after this gets merged.

Reply:
I have added this back in, and it will still run with unity Lewis number but will print an error also